core/services/vrf/integration_v2_test.go

```diff
@@ -875,6 +882,179 @@ func TestFulfillmentCost(t *testing.T) {
 	assert.Less(t, estimate, uint64(500000))
 }
 
+func TestStartingCountsV1(t *testing.T) {
```

Comment:
This test had to be moved here and become a bit more involved because foreign key constraints were failing when inserting rows into the `eth_tx_attempts` and `eth_receipts` tables, presumably due to txdb and those rows not actually getting committed.

There was also an import cycle after importing `heavyweight` in the `delegate_test.go` file so I've moved the test here to alleviate that.